documentation/modules/mtv-overview-page.adoc

```diff
@@ -28,10 +55,10 @@ The *Overview* page displays the following information:
 ** Succeeded
 ** Canceled
 
-* Operator: The namespace on which the {project-short} Operator is deployed and the status of the Operator.
+* Chart showing the number of running, failed, and succeeded migrations performed using {project-short} for each of the last 7 days
```

Comment:
even to me (and I saw demos of this screen couple of times) the difference between `migrations` here and `virtual machine migrations` below is not clear, does this line refer to migration plans? @yaacov

Reply:
@ahadas @yaacov  If a migration migrated 5 machines, wouldn't that add 1 to the migration count and 5 to the VM migration count? Since we have customers migrating tens or hundreds of machines in a single migration, I figured that's what was being counted here. 

I can add an example once we clarify the point.

Reply:
@ahadas I added this note:
[NOTE]
====
Since a single migration might involve many virtual machines, the number of migrations performed by {project-short} might vary significantly from the number of virtual machines that have been migrated by {project-short}.
====

If that's correct, I don't think we need an example, but up to you.

Reply:
@RichardHoch ack, this way (with the note) the term `migration` here is correlated with a migration-plan that can involve multiple VMs and then it makes sense - cool